dcos/marathon.py

```diff
@@ -753,7 +774,14 @@ def update_pod(self, pod_id, pod_json, force=False):
         :rtype: None
         """
 
-        return self._update('pods', pod_id, pod_json, force)
+        response = self._update_req('pods', pod_id, pod_json, force)
+        header_name = 'Marathon-Deployment-Id'
+
+        try:
+            return response.headers[header_name]
+        except KeyError:
+            template = 'Error: missing "{}" header from Marathon response'
```

Comment:
just a style thing- we havent been using that style. we only do try/except in cases where we can't guarantee there will be no exceptions (ie reading a file)